packages/gensx/src/utils/bundler.ts

```diff
@@ -101,6 +101,7 @@ export function getRollupConfig(
         jsx: "react-jsx",
         jsxImportSource: "@gensx/core",
         tsconfig: "./tsconfig.json",
+        module: "NodeNext",
```

Comment:
@jmoseley - i kept getting errors saying we needed to specify NodeNex here so i added it in; not sure if it's okay....perhaps a moot point since you're changing the bundling anyways